src/oop/special_class_method/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-special_class_method-medium-q2
+
+## Question Text
+
+You are about to start writing a program that involves luminous objects. Make a Java class called `Lamp` with the 
+following description.
+
+### Question Type:
+
+- Java question
+
+### Lamp Class:
+
+You are already given the following items:
+
+- Attributes:
+    - a `double` called **brightness** that is not accessible from other classes.
+    - a `boolean` called **isOn** that is not accessible from other classes.
+
+- Constructor:
+    - a `Lamp` constructor that is visible to other classes, accepts a `double` and uses it to initialize the
+      **brightness** attribute, and initializes the **isOn** attribute to true.
+
+You need to add the following items to the class:
+
+- Constructor:
+    - a `Lamp` constructor that is visible to other classes and accepts a `double` and a `boolean` and uses it to
+      initialize the **brightness** and **isOn** attributes.
```

Comment:
I think we can actually just give them this constructor. We really just care about testing the getter, setter, and toString methods in the medium category, and if we use this constructor in the tests, it could cause the test to fail because of the constructor and not what we are really concerned about in this category.

Reply:
It is only a medium question